src/renderer/components/Map.tsx

```diff
@@ -8,6 +8,8 @@ import { useTranslation } from 'react-i18next';
 
 import { EmptyClasses } from '../constants/animalsClasses';
 import AnimalsListTooltipContent from './AnimalsListTooltipContent';
+import Marker from '../../../assets/graphics/map-marker.svg';
```

Comment:
Let's keep the imports organized alphabetically (use Shift + Alt + O in VSCode).

Reply:
Done :)